apps/vessel-history/nginx.conf

```diff
@@ -31,7 +31,7 @@ http {
         listen ${PORT};
         root /usr/share/nginx/www/;
 
-        location ~* ^/map/(.+\..+)$ {
+        location ~* ^/vessel-viewer/(.+\..+)$ {
```

Comment:
ouch, sorry for this! 
my fault copying and pasting from fishing-map

Reply:
No problem 👍